compiler/rustc_resolve/src/build_reduced_graph.rs

```diff
@@ -1029,9 +1029,9 @@ impl<'a, 'b, 'tcx> BuildReducedGraphVisitor<'a, 'b, 'tcx> {
             }
         }
 
-        let macro_use_import = |this: &Self, span| {
+        let macro_use_import = |this: &Self, span: Span, warn_not_exported: bool| {
```

Comment:
```suggestion
        let macro_use_import = |this: &Self, span, warn_not_exported| {
```
The type annotations look redundant here.